backend/kafka_backend.go

```diff
@@ -0,0 +1,114 @@
+package backend
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/skroutz/downloader/job"
+)
+
+var eventsWG sync.WaitGroup
+
+// KafkaBackend notifies about a job completion by producing to a Kafka topic.
+type KafkaBackend struct {
+	producer *kafka.Producer
+	reports  chan job.CallbackInfo
+}
+
+// ID returns the identifier code for the respective backend
+// which is "kafka".
+func (b *KafkaBackend) ID() string {
+	return "kafka"
+}
+
+// Start sets up a kafka producer and a channel for delivery reports.
+func (b *KafkaBackend) Start(ctx context.Context, cfg map[string]interface{}) error {
+	var (
+		err         error
+		kafkaConfig kafka.ConfigMap
+	)
+
+	kafkaConfig = make(kafka.ConfigMap)
```

Comment:
You could do:

```go
cfg := make(kafka.ConfigMap)
```

This way you get rid of the variable declaration at the top.